src/traces/histogram/defaults.js

```diff
@@ -26,6 +27,9 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
     var x = coerce('x'),
         y = coerce('y');
 
+    var handleCalendarDefaults = Registry.getComponentMethod('calendars', 'handleTraceDefaults');
+    handleCalendarDefaults(traceIn, traceOut, ['x', 'y'], layout);
+
```

Comment:
previously we coerced calendars after the [return on no data](https://github.com/plotly/plotly.js/pull/1230/files?diff=split#diff-995f21c0a277c39bc47f02f65846d42aR40) - was it intentional to move this up?